test/e2e/helpers_test.go

```diff
@@ -523,6 +523,55 @@ func verifyEnterpriseWithPostgres(ctx context.Context, t *testing.T, env environ
 	require.Equal(t, http.StatusCreated, resp.StatusCode, fmt.Sprintf("STATUS=(%s), BODY=(%s)", resp.Status, string(body)))
 }
 
+// licenseOutput is the license section of the admin API root response.
+type licenseOutput struct {
+	License struct {
+		Customer   string `json:"customer"`
+		Dataplanes string `json:"dataplanes"`
+		Creation   string `json:"license_creation_date"`
+		Seats      string `json:"admin_seats"`
+		Product    string `json:"product_subscription"`
+		Plan       string `json:"support_plan"`
+		Expiration string `json:"license_expiration_date"`
+	} `json:"license"`
+}
+
+func getLicense(ctx context.Context, env environments.Environment, adminPassword string) (licenseOutput, error) {
```

Comment:
```suggestion
func getLicenseFromAdminAPI(ctx context.Context, env environments.Environment, adminPassword string) (licenseOutput, error) {
```
As we're in a winder context of all E2E tests I think it might make sense to name this unambiguously.